JSDemos/Demos/Scheduler/CurrentTimeIndicator/Angular/app/app.component.html

```diff
@@ -17,12 +17,17 @@
   <dxi-resource fieldExpr="movieId" [dataSource]="moviesData"> </dxi-resource>
 
   <div *dxTemplate="let model of 'appointment-template'">
-    <div class="movie">
-      <img [src]="getMovieById(model.appointmentData.movieId).image" />
-      <div class="movie-text">{{
-        getMovieById(model.appointmentData.movieId).text
-      }}</div>
-    </div>
+    <ng-container
+      *ngIf="
+        getMovieById
+          | apply : model.appointmentData.movieId : moviesData as movie
+      "
```

Comment:
Let's fix formatting here:

```suggestion
      *ngIf="(getMovieById | apply : model.appointmentData.movieId) as movie"
```